tpdatasrc/co8fixes/scr/expand_encounters.py

```diff
@@ -0,0 +1,29 @@
+from random_encounter import *
+import os
+	
+#name convention for encounterpacks is EncounterPack + Creator name + pack number
+#the file must contain a function called EncounterExpander which takes 2 parameters and returns a list of RE_entry to be added
+
+	
+def encounter_coordinator( setup, encounter ):
+	var = encounter_exists(setup, encounter) 						#original function is called immediately
+	if(get_f('qs_is_repeatable_encounter') and game.global_flags[500] == 1):		#checks if the encounter is a random and NC is enabled
+		extra_encounters(setup, encounter)						#therefore avoids interrupting scripted and special encounters
+	return var						
+		
+def extra_encounters(setup, encounter):
+	expanded_list = []
+	
+	encounterpacks = os.listdir('.\data\scr')	#correct location for the final release (?)

```

Comment:
this will re-run everytime it checks for encounters. is that intended? shouldn't this actually be only done once?
in addition, we need to use TIO functions here since they should also use files from .dat files, not only files that are unpacked on disk.
could the result of this function be cached?